services/web/server/src/simcore_service_webserver/projects/models.py

```diff
@@ -22,6 +29,38 @@ def to_project_type_db(cls, api_type: "ProjectTypeAPI") -> ProjectType | None:
         }[api_type]
 
 
+class ProjectDB(BaseModel):
```

Comment:
TIP: if this is a one-to-one with the sqlalchemy model, I would suggest to add a defensive check as
```python
assert set(ProjectDB.__fields__.keys()).issubset({c.name for c in projects.columns}) # nosec
```

Reply:
thanks, added